docs/openapi/components/schemas/common/SIMASteelImportLicense.yml

```diff
@@ -16,13 +16,7 @@ properties:
       type: string
       enum:
         - SIMASteelImportLicense
-  licenseNumber:
```

Comment:
We can't remove this, they are based on SIMA's documentation.

Reply:
that means it should be in the example, right?